src/encoded/static/libs/bootstrap/tooltip.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+// This module lets you have a tooltip pop up when any component is hovered over or focused. The
+// general usage is:
+//
+// <Tooltip trigger={<Trigger />} tooltipId="html-id-unique-on-page" css="custom-css-classes">
```

Comment:
Why not call "tooltipId" simply "id"?

Personal Preference Question: Instead of passing a component into another component, why not just pass a id of another element and read off the element or set the child of Tooltip as the target?

Reply:
This is a React-recommended way to do this (https://reactjs.org/docs/components-and-props.html#composing-components). Works well (used it a lot in ClinGen) and is simple and straightforward. The child of the tooltip is already the content of the tooltip. We could add a `TooltipTrigger` component as a required child, sort of like `<Param>`, but then we’d have to check that exactly one exists, then render it explicitly. This way is much simpler, which is probably why the React team recommends it.

I don‘t typically call these kinds of properties `id` in case someone confuses it with an HTML id, or in case we need more than one.

Reply:
@forresttanaka This is not big deal. Still, out of curiosity, I will stop by tomorrow. I think what I think this means is not what it does mean.

Reply:
I really worry something like shopping cart will take _weeks_ to code review over things like this that in the end change nothing.